gemrb/core/Scriptable/Actor.cpp

```diff
@@ -3338,7 +3338,7 @@ int Actor::NewStat(unsigned int StatIndex, ieDword ModifierValue, ieDword Modifi
 	return Modified[StatIndex] - oldmod;
 }
 
-int Actor::NewBase(unsigned int StatIndex, ieDword ModifierValue, ieDword ModifierType)
+int Actor::NewBase(unsigned int StatIndex, ieDwordSigned ModifierValue, ieDword ModifierType)
```

Comment:
The stats are words, so while this can solve the warnings, it's not really correct. It'd be better to add a new modifier type, eg. MOD_ADDITIVE_NEGATIVE and pass positive values. Now if there are legitimate high values of the stats, they would overflow.

Reply:
Ok.

Reply:
Done rework it